### ISSUE TITLE
Request to publish to npm

### ISSUE THREAD

**Issue by hafizMDM** [YenePay/yenepay.sdk.react-native#1]
Hello. 

Is this package going to be hosted to npm?

**Comment by SisayKelkle** [comment]
@hafizMDM Yes we are planning to publish it on npm once we clear out the initial bugs. You are welcome to help by the way.

**Comment by hafizMDM** [comment]
@SisayKelkle Cool. I will contribute.
